backend/mora/common.py

```diff
@@ -154,6 +154,12 @@ def update_payload(
         if field_tuple.type == mapping.FieldTypes.ADAPTED_ZERO_TO_MANY:
             # 'Fake' zero-to-one relation. Merge into existing list.
             updated_props = _merge_obj_effects(props, vals)
+            for p in updated_props:
```

Comment:
This only handles the "empty" payloads for `ADAPTED_ZERO_TO_MANY` changes. We never want empty payloads, right? I think it would make sense to move this outside of the if/else structure, so we perform this step on _every_ update payload we create, regardless of field type.

Reply:
This change is specific to the handling of relations. I don't think we have the same issue with attributes or states.

Byt you're right, I should check.

Reply:
Field type in this case is a bit abstract, and refers to the "types" of cardinalities we work with in MO, towards LoRa - so the different branches in the function (zero-to-one, zero-to-many, adapted-zero-to-many) all potentially refer to relations.

Reply:
So this function _only_ applies to relations?

I tried setting a break point in the other branches, and the test suite never tries to send empty payloads in those, only for the adapted-zero-to-many. (Maybe these are handled differently by MO.)

But if this function indeed only applies to relations, it will be both safe and correct to replace all empty payloads with ones with uuid = urn   ''.

Reply:
It applies to attributes and states as well, but we don't want those to be empty as well either, right? Actually, after having thought about it, maybe this fix belongs somewhere else, so we don't accidentally set `uuid` and `urn` in case we happen to be working with a "tilstand" where we forgot to set `gyldighed`.

Reply:
I will look into this, but the case of "empty" periods seem only to ever occur for relations.

Reply:
That's true, but it doesn't change the fact that this function is meant to work on attributes and states, as well as relations, so _if_ we ever happen to have "empty" periods for any of those, we'll have another problem on our hands.